playbooks/roles/katello_provisioning/tasks/katello.yml

```diff
@@ -0,0 +1,92 @@
+---
+# Product
+- name: 'find product'
+  shell: >
+    {{ katello_provisioning_hammer }} product info --name "CentOS" --organization "{{ katello_provisioning_organization }}"
+  register: katello_provisioning_product
+  ignore_errors: True
+
+- name: 'create centos product'
+  shell: >
+    {{ katello_provisioning_hammer }} product create
+    --name "CentOS"
+    --organization "{{ katello_provisioning_organization }}"
+  when: katello_provisioning_product.stderr.find('not found') != -1
+
+# Repo
+- name: 'find repo'
+  shell: >
+    {{ katello_provisioning_hammer }} repository info --name "CentOS 7" --product "CentOS" --organization "{{ katello_provisioning_organization }}"
```

Comment:
Nitpick -- I like the stacking method from https://github.com/Katello/forklift/pull/295/files#diff-4fc9c0aa4c54f421837dada92f7c1c01R117 for readability